src/server/game/Warden/WardenCheckMgr.h

```diff
@@ -31,32 +32,74 @@ enum WardenActions : uint8
     WARDEN_ACTION_BAN // TITLE Ban
 };
 
+// EnumUtils: DESCRIBE THIS
+enum WardenCheckCategory : uint8
+{
+    INJECT_CHECK_CATEGORY = 0, // checks that test whether the client's execution has been interfered with
+    LUA_CHECK_CATEGORY,        // checks that test whether the lua sandbox has been modified
+    MODDED_CHECK_CATEGORY,     // checks that test whether the client has been modified
+
+    NUM_CHECK_CATEGORIES // SKIP
+};
+
 // EnumUtils: DESCRIBE THIS
 enum WardenCheckType : uint8
 {
-    MEM_CHECK = 0xF3, // 243: byte moduleNameIndex + uint Offset + byte Len (check to ensure memory isn't modified)
-    PAGE_CHECK_A = 0xB2, // 178: uint Seed + byte[20] SHA1 + uint Addr + byte Len (scans all pages for specified hash)
-    PAGE_CHECK_B = 0xBF, // 191: uint Seed + byte[20] SHA1 + uint Addr + byte Len (scans only pages starts with MZ+PE headers for specified hash)
-    MPQ_CHECK = 0x98, // 152: byte fileNameIndex (check to ensure MPQ file isn't modified)
-    LUA_STR_CHECK = 0x8B, // 139: byte luaNameIndex (check to ensure LUA string isn't used)
-    DRIVER_CHECK = 0x71, // 113: uint Seed + byte[20] SHA1 + byte driverNameIndex (check to ensure driver isn't loaded)
-    TIMING_CHECK = 0x57, //  87: empty (check to ensure GetTickCount() isn't detoured)
-    PROC_CHECK = 0x7E, // 126: uint Seed + byte[20] SHA1 + byte moluleNameIndex + byte procNameIndex + uint Offset + byte Len (check to ensure proc isn't detoured)
-    MODULE_CHECK = 0xD9  // 217: uint Seed + byte[20] SHA1 (check to ensure module isn't injected)
+    NONE_CHECK     =   0, // SKIP
+    TIMING_CHECK   =  87, // nyi
```

Comment:
why nyi, it's always sent (don't ask me why). it's an empty payload for the request

Reply:
the result is not processed

Reply:
it is (or was)

Reply:
The actual timing result is never used, just logged.